tests/custom_resp_header_spec.rs

```diff
@@ -37,4 +32,17 @@ mod integration_tests {
     assert_eq!(response_headers.get("x-id").unwrap(), "1");
     assert_eq!(response_headers.get("x-value").unwrap(), "value");
   }
+
+  #[tokio::test]
+  async fn test_verify_response_headers() {
+    let schema_path = "tests/graphql_mock/test-custom-headers.graphql";
+    verify_response_headers(schema_path).await;
+  }
+
+  #[tokio::test]
+  async fn test_verify_response_headers_over_http() {
+    let schema_path =
+      "https://raw.githubusercontent.com/tailcallhq/tailcall/main/tests/graphql_mock/test-custom-headers.graphql";
+    verify_response_headers(schema_path).await;
```

Comment:
so the test could fail in case of disabled internet?
that could be unexpected, lets at least add some comment about this in code

Reply:
without internet connection, several cases in graphql_spec.rs will fail as well. We can not really test this offline